src/testsuite/arith128_test_char.c

```diff
@@ -0,0 +1,298 @@
+/*
+   arith128_test_char.c
+  
+    Created on: Oct 25, 2017
+        Author: sjmunroe
+  */
+
+
+#define __STDC_WANT_DEC_FP__    1
+
+#include <stdint.h>
+#include <stdio.h>
+#if 0
+#include <dfp/fenv.h>
+#include <dfp/float.h>
+#include <dfp/math.h>
+#else
+#include <fenv.h>
+#include <float.h>
+#include <math.h>
+#endif
+
+//#define __DEBUG_PRINT__
+
+#include "arith128.h"
+#include <testsuite/arith128_print.h>
+#include <vec_char_ppc.h>
+
+#include <testsuite/arith128_test_char.h>
+
+/*
+ * Return a vector boolean char with a true indicator for any character
+ * that is either Lower Case Alpha ASCII or Upper Case ASCII.
```

Comment:
Should `Alpha` also apply to `Upper Case` ?  (or is `Lower Case Alpha` actually redundant, as a character has to be alpha to have a case?)

Reply:
I was being pedantic. it should probably say "Lower Case ASCII or Upper Case ASCII" (including "Alpha" is unnecessary. moving on...